man/man5/puppet.conf.5

```diff
@@ -107,7 +107,7 @@ If a custom policy executable is configured, the CA puppet master will run it ev
 If a certificate request is not autosigned, it will persist for review\. An admin user can use the \fBpuppet cert sign\fR command to manually sign it, or can delete the request\.
 .
 .P
-For info on autosign configuration files, see the guide to Puppet\'s config files \fIhttp://docs\.puppetlabs\.com/guides/configuring\.html\fR\.
+For info on autosign configuration files, see the guide to Puppet\'s config files \fIhttps://docs\.puppetlabs\.com/guides/configuring\.html\fR\.
```

Comment:
But also, this man page should be automatically generated... I think? I'm not sure what causes it to get updated, anymore, but it's supposed to contain the same content as the setting descriptions in lib/puppet/defaults.rb.